spec/datetimeformat.html

```diff
@@ -603,20 +599,107 @@ <h1>Intl.DateTimeFormat.prototype.formatToParts ( _date_ )</h1>
       <h1>Intl.DateTimeFormat.prototype.resolvedOptions ()</h1>
 
       <p>
-        This function provides access to the locale and formatting options computed during initialization of the object. This function initially invokes the internal algorithm UnwrapDateTimeFormat to get the %DateTimeFormat% object on which to operate.
+        This function provides access to the locale and formatting options computed during initialization of the object.
       </p>
 
-      <p>
-        The function returns a new object whose properties and attributes are set as if constructed by an object literal assigning to each of the following properties the value of the corresponding internal slot of this DateTimeFormat object (see <emu-xref href="#sec-properties-of-intl-datetimeformat-instances"></emu-xref>): locale, calendar, numberingSystem, timeZone, hourCycle, weekday, era, year, month, day, hour, minute, second, and timeZoneName. Properties whose corresponding internal slots have the value *undefined* are not assigned.
-      </p>
+      <emu-alg>
+        1. Let _dtf_ be *this* value.
+        1. If Type(_dtf_) is not Object, throw a *TypeError* exception.
+        1. Let _dtf_ be ? UnwrapDateTimeFormat(_dtf_).
```

Comment:
Good catch; the previous specification was wrong here. This is an observable change; would you be interested in upstreaming a test262 test for it? Here's a one-liner assertion: `"string" === typeof Intl.DateTimeFormat.prototype.resolvedOptions.call(Intl.DateTimeFormat.call({})).locale`

Reply:
In how far was the previous specification wrong?

Reply:
In the sense that, previously, there was no call to UnwrapDateTimeFormat, and now there is.

Reply:
The requirement to call UnwrapDateTimeFormat was placed in the description of `resolvedOptions()`:

https://tc39.github.io/ecma402/#sec-intl.datetimeformat.prototype.resolvedoptions (emphasis mine):
> This function provides access to the locale and formatting options computed during initialization of the object. **This function initially invokes the internal algorithm UnwrapDateTimeFormat to get the %DateTimeFormat% object on which to operate.**

Reply:
Oh, right, no change then.